### DISCUSSION TITLE
[Request] operating system targetting

### DISCUSSION THREAD

**Discussion by aidrouge** [giuliopons/adadmin#42] [Ideas]
hello can you add os in you next update in targeting

**Comment by giuliopons** [comment]
Ok. I will add this feature in next release.

**Comment by giuliopons** [comment]
One question more. You need to detect IOS/ANDROID on mobile? Or also windows/mac on desktop?

**Comment by aidrouge** [comment]
Those will be a start some tools even target os version like android 9 android 10 ect but those are enough for now. Thank you.
Ios 
android
Windows 
os x
linux 
ubunto 
chrome os

**Comment by giuliopons** [comment]
Hi aidrouge,
I think that with this new feature the desktop/device targeting is a bit confusing now. What do you think?

![image](https://user-images.githubusercontent.com/1871627/200166350-a1a03c2c-c583-4900-b062-75a386795095.png)

Should they coexist?

**Comment by aidrouge** [comment]
If mobile & desktops not selected 
Then show all os
Else if mobile selected
Then show only mobile os
If desktop selected show only desktop os
